THIRD_PARTY_README

```diff
@@ -3043,14 +3043,14 @@ performance, or use of this material.
 
 -------------------------------------------------------------------------------
 
-%% This notice is provided with respect to zlib v1.2.13, which may be included 
+%% This notice is provided with respect to zlib v1.3.1, which may be included 
```

Comment:
There is a trailing space after `included`. Please remove.

Reply:
Done.  There is trailing whitespace throughout the THIRD_PARTY_README files, which I left alone, but given that I was changing this line anyway, it is a good opportunity to remove this occurrence.